lib/auto_html/auto_html_for.rb

```diff
@@ -22,31 +22,30 @@ def auto_html_for(raw_attrs, &proc)
         return unless ActiveRecord::Base.connection.table_exists? self.table_name
       end
 
-      suffix =  AutoHtmlFor.auto_html_for_options[:htmlized_attribute_suffix]
-      auto_html_for_columns = [raw_attrs].flatten.map { |a| "#{a}#{suffix}" }
-      
+      suffix = AutoHtmlFor.auto_html_for_options[:htmlized_attribute_suffix]
+      auto_html_for_columns = [raw_attrs].flat_map{ |a| "#{a}#{suffix}" }
+
       # Needed for Mongoid
       column_names = self.respond_to?(:column_names) ? self.column_names : fields.keys
-      
-      missing_cache_columns =  auto_html_for_columns - column_names
+
+      missing_cache_columns = auto_html_for_columns - column_names
       missing_cache_columns.each do |missing_cache_column|
-        raw_attr = missing_cache_column.gsub(suffix, '')
+        raw_attr = missing_cache_column.sub(suffix, '')
         define_method(missing_cache_column) do
           val = self[raw_attr] || self.send(raw_attr.to_sym)
           auto_html(val, &proc)
         end
       end
-      
+
       cache_columns = auto_html_for_columns - missing_cache_columns
       cache_columns.each do |cache_column|
-        raw_attr = cache_column.gsub(suffix, '')
+        raw_attr = cache_column.tr(suffix, '')
```

Comment:
In [line 33](https://github.com/dointeractive/auto_html/pull/9/files#diff-cb3a05028f57708bd178f843b0e50ad7R33) you've replaced `gsub` with `sub`, and in this line you've used `tr`.
Is there any reason?